source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOTypeaheadDebouncer.mm

```diff
@@ -0,0 +1,38 @@
+//
+//  ACOTypeaheadDebouncer.mm
+//  AdaptiveCards
+//
+//  Created by Jyoti Kukreja on 25/01/23.
+//  Copyright © 2023 Microsoft. All rights reserved.
+//
+
+#import "ACOTypeaheadDebouncer.h"
+
+@interface ACOTypeaheadDebouncer ()
+@property (nonatomic) NSTimeInterval delay;
+@end
+
+@implementation ACOTypeaheadDebouncer
+
+- (instancetype)initWithDelay:(NSTimeInterval)delay
```

Comment:
you need to run clang-format to fix the formating error. let me know if you need help in that.